WalletWasabi.Gui/Converters/FeeTargetTimeConverter.cs

```diff
@@ -13,16 +13,16 @@ public object Convert(object value, Type targetType, object parameter, CultureIn
 		{
 			if (value is int feeTarget)
 			{
-				if (feeTarget >= Constants.TwentyMinutesConfirmationTarget && feeTarget <= 6) // minutes
+				if (feeTarget is >= Constants.TwentyMinutesConfirmationTarget and <= 6) // minutes
```

Comment:
Hmm, you can compare with pattern matching operator?

Reply:
Sorry I don't get the question.

Reply:
Nevermind I tested it - it is OK. Consider playing with .NET fiddle to test and verify basic functionality, it is easy to use and a lot can be learned.